src/components/Docs/SnippetViewer/WriteRequestViewer.tsx

```diff
@@ -67,13 +72,12 @@ ${
       const writeTuples = opts.relationshipTuples
         ? opts.relationshipTuples
             .map(
-              ({ user, relation, object, _description }) => `
-      ${
-        _description ? `// ${_description}\n      ` : ''
-      }{ user: '${user}', relation: '${relation}', object: '${object}'}`,
+              (tuple) => `
+      ${tuple._description ? `// ${tuple._description}\n      ` : ''}${JSON.stringify(tuple)}`,
```

Comment:
This is not meant to be JSON.

While it works, it's rarely that code is written this way

Reply:
What's your suggestion for the alternative?

Reply:
{
  // description
  user: "abc:abc",
  relation: "abc",
  object: "abc:abc",
  context: whatever
}

Reply:
Also you need to exclude the description from the stringified version